modules/images/src/lib/utils/assert.ts

```diff
@@ -0,0 +1,5 @@
+export function assert(condition: any, message?: string): void {
```

Comment:
Can we use assert from 
`import {assert} from '@loaders.gl/loader-utils';` ?

Reply:
Yes initially I tried to avoid that dependency in this module (deck.gl is always worried about bundle size impact) but since we are depending on it now that makes sense.